### ISSUE TITLE
Tensor lambda_input:0, specified in either feed_devices or fetch_devices was not found in the Graph

### ISSUE THREAD

**Issue by lkuich** [migueldeicaza/TensorFlowSharp#440]
**Describe the bug**
My graph is trained in Python with Tensorflow 1.12 and uses [tf.keras.layers.Lambda](https://www.tensorflow.org/api_docs/python/tf/keras/layers/Lambda) as it's input layer, named `lambda_input`. When running it with an image input tensor with TensorFlowSharp 1.12 in .NET Core 2.2, I get the following error:
"Tensor lambda_input:0, specified in either feed_devices or fetch_devices was not found in the Graph"

Likewise if I just skip the input tensor, I get "Tensor output/Softmax:0, specified in either feed_devices or fetch_devices was not found in the Graph"

**To Reproduce**
[bug.zip](https://github.com/migueldeicaza/TensorFlowSharp/files/3388533/bug.zip)

I've appended my model and a simple script that reproduces the issue. It occurs when I run the runner, regardless if there's input or not:
```csharp
var graphBytes = File.ReadAllBytes("frozen.pb");
var graph = new TFGraph();
graph.Import(graphBytes);

var runner = new TFSession().GetRunner();
var gInput = graph["lambda_input"];
var gResult = graph["output/Softmax"];

// runner.AddInput(gInput[0], myImageTensor);
runner.Fetch(gResult[0]);

var result = runner.Run()[0];
```

**Expected behavior**
The model should run normally without error.

**Screenshots**
![image](https://user-images.githubusercontent.com/7741982/61168162-a6e22e00-a4fe-11e9-82a3-9ea6bf7e0794.png)

**Desktop (please complete the following information):**
 - OS: Windows 10

**Additional context**
Are lambda layers supported by TensorflowSharp? I know they're a problem in Tensorflow.js.

Thank you,
Loren K

**Comment by lkuich** [comment]
Just noticed, I failed to pass the graph to the session.

Should look like this:
```csharp
var graphBytes = File.ReadAllBytes("frozen.pb");
var graph = new TFGraph();
graph.Import(graphBytes);

var runner = new TFSession(graph).GetRunner();
var gInput = graph["lambda_input"];
var gResult = graph["output/Softmax"];

// runner.AddInput(gInput[0], myImageTensor);
runner.Fetch(gResult[0]);

var result = runner.Run()[0];
```
